lib/wallet/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -1428,18 +1432,18 @@ instance FromJSON SchemaApiErrorCode where
 -------------------------------------------------------------------------------}
 
 -- Dummy instances
-instance EncodeAddress ('Testnet 0) where
+instance {-# INCOHERENT #-} EncodeAddress ('Testnet 0) where
```

Comment:
Should add note and link to a tech dept ticket regarding removing the orphan instances.